### DISCUSSION TITLE
Furk.net service request

### DISCUSSION THREAD

**Discussion by sasagr** [itsToggle/plex_debrid#211] [Ideas]
Hey. I have been wanting to ask you if you would be interested to add a new "debrid" service. Probably less popular then RD or PM but I think still used among some users, especially in the Kodi world.
I m talking about Furk.net. I bought a lifetime subscription about 10 years ago and still using it as much as I can.
Furk has its own "download proxy", that is a basic search of all kind of files (including a big amount of video of course). When you make a search you get results of torrent that are already cached or not. Those cached are available for download/stream otherwise they can be put in a queue of downloading/caching. You will be notified by email when it is cached and ready for download. There is limit of what you can cache and not download eventually. So no sense to add to cache if you won't download/stream. You can also add cached files to your "My Files" where all your files are collected. Those files could be still deleted by the server unless you protect them but if I remember correct you cannot protect more than 20 Gb. Other number to keep in mind is that there is a limit of bandwidth in download/stream. The lifetime gives max 250 Gb/month, so not so much if you want to watch 4k movies every night. Unfortunately, they don't give access to "My Files" with FTP or WebDAV but the good part of it is that they have a public API so it is possible to access "My Files" from the API.
So I think it could be interesting as first step to be able to create a rclone backend for it and be able to mount the "My Files" archive with files ready to me played. I asked a few years ago in the rclone community but nothing happened so far.
The second part could be to use it as a scraper in addition to RD, PM,... so Furk could search the Movie/Show for you and make it ready in My Files, cached or not yet cached. You will not need to use any torrent indexer, Furk does it all.
Let me know what you think and if you have any questions. I also understand if this is not your priority at the moment. If you don't have an account I can give you an invite code or even better share with you my account just for development purposes.

**Comment by itsToggle** [comment]
That sounds like a cool service! The limitations are pretty heavy, but there are definitly some benefits to a service like this one. Ill see when Ill find the time to play around with this, its probaply a lower priority issue for now though.
In the meantime, you could also try to write an rclone backend yourself, there is some good documentation for that on the official rclone channels :)

**Reply by sasagr** [reply]
I will try .. maybe you can suggest me a good example of a simple one with api

**Comment by sasagr** [comment]
I gave a look at the rclone documentation and it seems a bit too much with my basic python skills. I m not sure I have the strength to start learning something new at the moment. I will prefer to wait in case there will be a chance for somebody else to do it. 
Thanks for the amazing job you are currently doing!
